src/Statistics/include/Statistics/Histogram.h

```diff
@@ -36,6 +36,12 @@ struct Histogram {
 // estimates the risk score using `HistogramRiskScore` and returns the histogram
 // which minimizes it. The histogram will not own the data.
 [[nodiscard]] std::optional<Histogram> BuildHistogram(absl::Span<const uint64_t> data);
+
+// The functions aids histogram rendering.
```

Comment:
I thought that this file and this module only dealt with the mathematical part of histograms, so this function feels out of place. What about moving it to `HistogramWidget.h`/`.cpp`. Similarly to how `ValueToAxisLocation` is there..

Reply:
You're right. I thought it makes sense to make the method public, so some future widget might use it to plot some different kind of histograms too. But I guess, it's too early.